framework/auth/oauth_scopes.py

```diff
@@ -41,6 +41,10 @@ class CoreScopes(object):
     APPLICATIONS_READ = 'applications_read'
     APPLICATIONS_WRITE = 'applications_write'
 
+    NODE_LOG_READ = 'nodes.logs_read'
+
+    NULL = 'null'
```

Comment:
Ah, I see: NULL is for views that would never allow write in principle, due to something totally outside the oauth scope machinery. 

Actually, I can see where that would be less confusing than the alternative- it indicates that the `require_read_scopes` entry is not applicable. I'm ok with that, as long as it is ONLY used in that context.